optuna_dashboard/ts/apiClient.ts

```diff
@@ -217,6 +217,24 @@ export const saveTrialNoteAPI = (
     })
 }
 
+export const tellTrialAPI = (
+  studyId: number,
+  trialId: number,
+  state: TrialState,
+  values?: string[]
+): Promise<void> => {
+  const req: { [name: string]: TrialState | string[] } = { state: state }
```

Comment:
Could you type the request like the following?

```
{
    state: TrialState
    values?: number[]
}
```

Reply:
Fixed.